infra/cifuzz/fuzz_target.py

```diff
@@ -72,7 +73,9 @@ def fuzz(self):
     command += [
         '-e', 'FUZZING_ENGINE=libfuzzer', '-e', 'SANITIZER=address', '-e',
         'RUN_FUZZER_MODE=interactive', 'gcr.io/oss-fuzz-base/base-runner',
-        'bash', '-c', 'run_fuzzer {0}'.format(self.target_name)
+        'bash', '-c', 'run_fuzzer {0} {1} {2}'.format(self.target_name,
```

Comment:
nit: let's use named arguments when there are many of them, e.g. `'{fuzz_target} {options}'.format(fuzz_target=self.target_name, options='-seed=1337 -len_control=0'`

Reply:
Fixed